services/clusters-keeper/src/simcore_service_clusters_keeper/modules/dask.py

```diff
@@ -22,15 +23,19 @@ async def ping_gateway(*, url: AnyUrl, password: SecretStr) -> bool:
             auth=basic_auth,
             asynchronous=True,
         ) as gateway:
-            cluster_reports = await asyncio.wait_for(gateway.list_clusters(), timeout=5)
-        _logger.info("found %s clusters", len(cluster_reports))
+            await asyncio.wait_for(gateway.list_clusters(), timeout=5)
         return True
     except asyncio.TimeoutError:
-        _logger.debug("gateway ping timed-out, it is still starting...")
-    except ClientError:
+        _logger.info(
+            "osparc-gateway %s ping timed-out, the machine is likely still starting...",
+            url,
+        )
+    except (ClientError, ValueError):
```

Comment:
Q: where might be this `ValueError` be raised from? `ValueError` normally denote "wrong arguments" but `list_cluster` has not arguments so `Gateway`?

Reply:
ask dask-gateway